### ISSUE TITLE
Consolidate common CV functions

### ISSUE THREAD

**Issue by atar13** [tritonuas/obcpp#143]
There's chunks of code in the CV files that are repeated.

These are the functions we should have in [src/cv/utilities.cpp](https://github.com/tritonuas/obcpp/blob/main/src/cv/utilities.cpp):

- [ ] openCVToTensor
  - Should be based on these lines in `matching.cpp`. @AchuthKrishna found a bug that he fixed so we should use his version https://github.com/tritonuas/obcpp/blob/main/src/cv/matching.cpp#L5-L28.
  https://github.com/tritonuas/obcpp/blob/3232fbd88398fd884e47d70add8d56f6b9a41f06/src/cv/matching.cpp#L29-L35 
- [ ] tensorToModelInput
  - Basically this https://github.com/tritonuas/obcpp/blob/main/src/cv/segmentation.cpp#L96-L99
- [ ] tensorToOpenCV
  - https://github.com/tritonuas/obcpp/blob/main/src/cv/segmentation.cpp#L101-L114
- [ ] loadPytorchModel
  - Take a model path string and return a `torch::jit::script::Module`
  - Basically like this code segment: https://github.com/tritonuas/obcpp/blob/cv-saliency-integration/src/cv/segmentation.cpp#L6-L12
